examples/sequence.rs

```diff
@@ -187,22 +183,20 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
     let seq2 = Sequence::new([Box::new(tween_move) as BoxedTweenable<_>, tracks.into()]);
 
     commands.spawn((
-        SpriteBundle {
-            sprite: Sprite {
-                color: BLUE.into(),
-                custom_size: Some(Vec2::new(size * 3., size)),
-                ..default()
-            },
-            ..Default::default()
+        Sprite {
+            color: BLUE.into(),
+            custom_size: Some(Vec2::new(size * 3., size)),
+            ..default()
         },
         BlueSprite,
         Animator::new(seq2),
     ));
 }
 
 fn update_text(
-    mut query_text_red: Query<&mut Text, (With<RedProgress>, Without<BlueProgress>)>,
-    mut query_text_blue: Query<&mut Text, (With<BlueProgress>, Without<RedProgress>)>,
+    query_text_red: Query<&Children, (With<RedProgress>, Without<BlueProgress>)>,
+    query_text_blue: Query<&Children, (With<BlueProgress>, Without<RedProgress>)>,
```

Comment:
You could be using `Single` on these two instead of `Query`, and I think the mutually exclusive filters are not needed anymore due to the change of mutability of the query. Just a minor and optional suggestion.